updated with most relevant certificate store address

bhushang19 opened the pull request:
application certificates most probably wont be installed on the backend machine inside the Current User location, instead they are installed in the Local Computer location.

bhushang19 committed:
updated with most relevant certificate store address

application certificates most probably wont be installed on the backend machine inside the Current User location, instead they are installed in the Local Computer location.

surajmb reviewed (changes requested):
Let's say current user or local computer since it can be at any location. As long as they are able to locate their own certificate.

bhushang19 committed:
Updated steps to open local machine cert store

Updated steps to open local machine cert store as mentioned here https://github.com/MicrosoftDocs/azure-docs/pull/56119#issuecomment-651285649_

bhushang19 reviewed (commented):
added steps to open local machine cert store using mmc.
